model/event/host_event.go

```diff
@@ -6,6 +6,7 @@ import (
 
 	"github.com/evergreen-ci/evergreen"
 	"github.com/evergreen-ci/evergreen/db"
+	_ "github.com/mongodb/amboy/registry" // bleh
```

Comment:
Why?

Reply:
my goimports was old and coughed up on this, but I think I fixed it.